app/css/vendors/ip-camera.css

```diff
@@ -0,0 +1,77 @@
+.camera-view {
+  display: flex;
+  flex-direction: column;
+}
+
+.camera-view__body {
```

Comment:
> update the existing styles

I'll check how much work it is, if not much - will do in this pr, otherwise - I'll handle this in the follow up to not mix things too much.